### PR TITLE
LightBurnSoftware.LightBurn version 1.3.00

### PR DESCRIPTION
**Pull request by mwalling**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91024)

### CODE REVIEW

**Comment by mwalling** [comment]
cc @dwittenberg since you have the majority of previous LB submissions.

**Comment by mwalling** [comment]
@microsoft-github-policy-service agree [company="{your company}"]

**Comment by mwalling** [comment]
@microsoft-github-policy-service agree

File: manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.5-1-22610-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LightBurnSoftware.LightBurn
+PackageVersion: 1.3.00
+PackageLocale: en-US
+Publisher: LightBurn Software
+PublisherUrl: https://lightburnsoftware.com/
+PublisherSupportUrl: https://forum.lightburnsoftware.com/
+# PrivacyUrl:
```
Comment (mdanish-kh) [inline manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml:10]:
```suggestion
PrivacyUrl: https://forum.lightburnsoftware.com/privacy
```
Additional metadata

File: manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Publisher: LightBurn Software
+PublisherUrl: https://lightburnsoftware.com/
+PublisherSupportUrl: https://forum.lightburnsoftware.com/
+# PrivacyUrl: 
+# Author: 
+PackageName: LightBurn
+PackageUrl: https://lightburnsoftware.com/
+License: Proprietary
+# LicenseUrl: 
+Copyright: Created by LightBurn Software, LLC
```
Comment (mdanish-kh) [inline manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml:16]:
```suggestion
Copyright: © 2022, LightBurn Software
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml (hunk 1 above)
Comment (mwalling) [reply manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml:10]: I don't think this is correct. That's the privacy policy for the community forum. I can't find one on the main product's website.

File: manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml (hunk 2 above)
Comment (mwalling) [reply manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml:16]: This does not align with what is in the Help->About window.

image

**Comment by mdanish-kh** [comment]
@msftbot reset feedback

**Review by mdanish-kh — APPROVED** [review]

File: manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml (hunk 1 above)
Comment (mdanish-kh) [reply manifests/l/LightBurnSoftware/LightBurn/1.3.00/LightBurnSoftware.LightBurn.locale.en-US.yaml:10]: Ah right, apologies

**Pull request merged**